src/scss/_main.scss

```diff
@@ -695,11 +691,27 @@ nav {
   margin-left: 3em;
 }
 
-.build-history .-build {
+.recent-build {
   position: relative;
 }
 
-.build-history .-build .-icon {
+.recent-build.-current > svg {
```

Comment:
css class targets inner svg to not interfere with the hover tooltip's z-axis placement